### DISCUSSION TITLE
admin panel voyager

### DISCUSSION THREAD

**Discussion by gcorp-llc** [HassanZahirnia/laravel-package-ocean#73] [Package Suggestions]
https://github.com/the-control-group/voyager

**Comment by HassanZahirnia** [comment]
Hey! Thanks for your suggestion.

I see your package has not been updated for a long time, and doesn't seem be compatible with Laravel 10. I would only add packages that are actively maintained and up to date with latest Laravel versions.

If you attend to updating it more often I'll be happy to add it to the database!
